src/platform/mhv/secondary-nav/tests/MhvSecondaryNav.unit.spec.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { expect } from 'chai';
+import { renderInReduxProvider } from '@department-of-veterans-affairs/platform-testing/react-testing-library-helpers';
+import MhvSecondaryNav from '../containers/MhvSecondaryNav';
+
+const stateFn = ({
+  loading,
+  mhvTransitionalMedicalRecordsLandingPage = false,
+  mhvIntegrationMedicalRecordsToPhase1 = false,
+} = {}) => ({
+  featureToggles: {
+    loading,
+    mhvTransitionalMedicalRecordsLandingPage,
+    mhvIntegrationMedicalRecordsToPhase1,
+  },
+});
+
+const setup = ({ initialState = stateFn() } = {}) =>
+  renderInReduxProvider(<MhvSecondaryNav />, { initialState });
+
+describe('<MhvSecondaryNav />', () => {
+  it('renders nothing when state.featureToggles is unset', () => {
+    const { container } = setup({ initialState: {} });
+    expect(container).to.be.empty;
+  });
+
+  it('renders nothing when loading features', () => {
+    const initialState = stateFn({ loading: true });
+    const { container } = setup({ initialState });
+    expect(container).to.be.empty;
+  });
+
+  it('renders nothing when loading is undefined', () => {
+    const initialState = stateFn({ loading: undefined });
+    const { container } = setup({ initialState });
+    expect(container).to.be.empty;
+  });
+
+  it('renders', () => {
+    const initialState = stateFn({ loading: false });
+    const { getByRole } = setup({ initialState });
+    getByRole('navigation', { name: 'My HealtheVet' });
+    const mrLink = getByRole('link', { name: /^My HealtheVet/ });
+    expect(mrLink.href).to.match(/my-health$/);
+  });
+
+  describe('Medical Records href', () => {
```

Comment:
I considered writing this section as below, but wasn't certain if it was easier to read and comprehend vs what's in the PR.

```js
    [
      {
        nextState: { mhvIntegrationMedicalRecordsToPhase1: true },
        href: '/my-health/medical-records',
      },
      {
        nextState: {
          mhvIntegrationMedicalRecordsToPhase1: true,
          mhvTransitionalMedicalRecordsLandingPage: true,
        },
        href: '/my-health/medical-records',
      },
      {
        nextState: { mhvTransitionalMedicalRecordsLandingPage: true },
        href: '/my-health/records',
      },
      {
        nextState: {
          mhvIntegrationMedicalRecordsToPhase1: false,
          mhvTransitionalMedicalRecordsLandingPage: false,
        },
        href: '/my-health/medical-records',
      },
    ].forEach(({ nextState, href }) => {
      const value = JSON.stringify(nextState);
      it(`when state is ${value}\n\t href is ${href}`, () => {
        const { getByRole } = setup({ initialState: stateFn(nextState) });
        const mrLink = getByRole('link', { name: /^Records/ });
        expect(mrLink.href).to.match(new RegExp(`${href}$`));
      });
    });
```